### PR TITLE
Updated Maki.gg

### PR DESCRIPTION
**Pull request by m5rian**
Added missing pages & Made presence more detailed

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix errors, bump version, make sure the presence follows all guidelines, upload proof of it working

**Comment by SlowLife1661** [comment]
Buttons are not allowed on home page

**Comment by M5rian** [comment]
Can I add them everywhere except the homepage?

**Comment by SlowLife1661** [comment]
Yes, *read the guidelines*

File: websites/M/Maki.gg/dist/metadata.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "nl": "Een multifunctionele bot met serverbeheer, nivellering, economie, beheertools, leuke opdrachten en meer met een intuïtief dashboard!"
   },
   "url": "maki.gg",
-  "version": "1.2.6",
+  "version": "1.3.0",
   "logo": "https://i.imgur.com/xzx4dEZ.gif",
```
Comment (SlowLife1661) [inline websites/M/Maki.gg/dist/metadata.json:13]:
```suggestion
  "logo": "https://i.imgur.com/xzx4dEZ.png",
```

**Comment by M5rian** [comment]
Oh, sorry I didn't read the guildlines carefully

**Comment by M5rian** [comment]
Can somebody help me with the CodeFactor? It says Autofix available, but I can't find out how to autofix my issues.

**Comment by SlowLife1661** [comment]
again read the guidelines promote the site with buttons is not allowed either

**Comment by SlowLife1661** [comment]
> Can somebody help me with the CodeFactor? It says Autofix available, but I can't find out how to autofix my issues.

just check on Codacy

**Comment by M5rian** [comment]
I’ll send a proof of it working later

File: websites/M/Maki.gg/presence.ts [hunk 2]
```diff
@@ -1,47 +1,117 @@
...
+  // Homepage
+  if (document.location.pathname === "/") {
+    presenceData.details = "On Homepage";
+
+    const excatPath = document.location.href.substring(document.location.host.length + document.location.protocol.length + 2);
+    if (excatPath === "/#features")
+      presenceData.state = "Looking at Makis features";
+  }
+  if (document.location.pathname !== "/") {
+    const [ButtonData, ButtonData] = {
```
Comment (Bas950) [inline websites/M/Maki.gg/presence.ts:23]: yea that wont work...

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by M5rian** [comment]
Here is a proof it it working. Do you guys need more than one image? I fixed the final things I saw and in my browser everything seems to work without errors. I'm sorry that acted a bit dumb, never made a pull request before and it was a bit confusing that I seem to miss a few things.

image

I hope you all are having a great day.
Thanks

**Comment by Quintenvw** [comment]
Original author and developer of Maki.gg here, I asked Marian to update this presence. It all seems to work and all web pages have been added to the presence.

File: websites/M/Maki.gg/presence.ts [hunk 3]
```diff
@@ -1,47 +1,110 @@
...
+      presenceData.details = "Reading";
+      presenceData.state = "The boring terms of service";
+    } else if (document.location.pathname === "/team") {
+      presenceData.details = "Looking at the";
+      presenceData.state = "Team";
     }
   }
 
-  if (presenceData.details == null) {
+  if (presenceData.details === null) {
```
Comment (Slowlife01) [inline websites/M/Maki.gg/presence.ts:105]:
```suggestion
  if (!presenceData.details) {
```

**Review by Slowlife01 — APPROVED** [review]

**Comment by M5rian** [comment]
Doesn't mean approved, that you accepted the changes? Oh no what have I done

**Comment by Bas950** [comment]
> Doesn't mean approved, that you accepted the changes? Oh no what have I done

Yea SlowLife accepted it, but you need 2 accepts to get it merged.

**Comment by M5rian** [comment]
oh, I see. I think I reopened it. I'm sorry

**Review by Bas950 — APPROVED** [review]

**Comment by M5rian** [comment]
😮 First pull request. Thank you guys.

**Pull request merged**